src/js/media-container.js

```diff
@@ -43,11 +44,6 @@ template.innerHTML = `
       pointer-events: auto;
     }
     
-    ::slotted([slot=poster]) {
```

Comment:
Does this not break the poster sizing to the full container?

Reply:
nah, it was repetitively redundant